spec/form_spec.cr

```diff
@@ -227,6 +227,29 @@ describe "LuckyRecord::Form" do
     end
   end
 
+  describe ".save!" do
+    context "on success" do
+      it "saves and returns the record" do
+        params = {"joined_at" => now_as_string, "name" => "New Name", "age" => "30"}
+
+        record = UserForm.save!(params)
+
+        record.is_a?(User).should be_true
+        record.name.should eq "New Name"
+      end
+    end
+
+    context "on failure" do
+      it "raises an exception" do
+        params = {"name" => "", "age" => "30"}
+
+        expect_raises(LuckyRecord::InvalidFormError) do
```

Comment:
You can pass a string as the second arg to `expect_raises` and it will check that it is included in the error message. So maybe it would be good to add "Invalid UserForm"